i18n/en-US.yml

```diff
@@ -67,8 +69,10 @@ actions:
     itineraryExistenceCheckFailed: Error checking whether your selected trip is possible.
     preferencesSaved: Your preferences have been saved.
     smsInvalidCode: The code you entered is invalid. Please try again.
-    smsResendThrottled: A verification SMS was sent to the indicated phone number less than a minute ago. Please try again in a few moments.
-    smsVerificationFailed: Your phone could not be verified. Perhaps the code you entered has expired. Please request a new code and try again.
+    smsResendThrottled: A verification SMS was sent to the indicated phone number
+      less than a minute ago. Please try again in a few moments.
+    smsVerificationFailed: Your phone could not be verified. Perhaps the code you
+      entered has expired. Please request a new code and try again.
```

Comment:
See above suggestion on large spaces ( which also applies to other changes in this PR)